Documentation/contributing.rst

```diff
@@ -181,6 +181,31 @@ If for some reason, running of the provisioning script fails, you should bring t
 
     $ vagrant halt
 
+Packer-CI-Build
+^^^^^^^^^^^^^^^^
+
+As part of Cilium development, we use a custom base box with a bunch of
+pre-installed libraries and tools that we need to enhance our daily workflow.
+That base box is built with `Packer <https://www.packer.io/>`_ and it is hosted
+in the `packer-ci-build <https://github.com/cilium/packer-ci-build>`_ Github
+repository.
+
+New versions of this box can be created via `Jenkins
+<https://jenkins.cilium.io/job/cilium/job/packer-ci-build/job/master/>`_, where
+new builds of the image will be pushed to  `Vagrant Cloud
+<https://app.vagrantup.com/cilium>`_ . The version of the image correspods to
+the `BUILD_ID <https://qa.nuxeo.org/jenkins/pipeline-syntax/globals#env>`_ on
+Jenkins job.  That version ID will be used in Cilium `Vagrantfiles
+<https://github.com/cilium/cilium/blob/master/test/Vagrantfile#L10>`_.
+
+Changes to this image are made via contributions to the packer-ci-build
+repository. Authorized GitHub users can trigger builds with a GitHub comment on
+the PR containing the trigger phrase ``build-me-please``.
+
+This box will need to be updated when a new developer needs a new dependency
+that it's not satisfied in the current version of the box, or the cilium
```

Comment:
... that is not installed in the current version of the box, or if a dependency that is cached within the box becomes stale.